src/components/Profile/components/PersonalInfoList/components/UpdatePhoneDialog/index.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import MaskedInput from 'react-text-mask';
+import EditIcon from '@material-ui/icons/Edit';
+import {
+  IconButton,
+  FormControl,
+  Input,
+  InputLabel,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import { userService, user } from 'services/user';
+import GordonSnackbar from 'components/Snackbar';
+import './index.css';
+
+const UpdatePhone = () => {
+  const [open, setOpen] = React.useState(false);
+  const [mobilePhoneNumber, setMobilePhoneNumber] = useState();
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSubmit = async () => {
+    await userService.setMobilePhoneNumber(mobilePhoneNumber);
+    handleClose();
+    handleChangeMobilePhoneNumber();
+  };
+
+  const createSnackbar = (message, severity) => {
+    setSnackbar({ message, severity, open: true });
+  };
+
+  const handleChangeMobilePhoneNumber = async (mobilePhoneNumber) => {
+    try {
+      await user.setMobilePhoneNumber(mobilePhoneNumber);
+      createSnackbar('Your phone number will update within a couple hours.', 'success');
+    } catch {
+      createSnackbar('Phone number failed to update. Please contact CTS.', 'error');
+    }
+  };
+
+  return (
+    <div className="gc360-updatephone-dialog">
+      <IconButton className="gc360-my-profile_edit-icon" onClick={handleClickOpen}>
+        <EditIcon style={{ fontSize: 20 }} />
+      </IconButton>
+      <Dialog open={open} onClose={handleClose}>
+        <DialogTitle className="gc360-updatephone-dialog_title">Update Phone Number</DialogTitle>
+        <DialogContent className="gc360-updatephone-dialog_content">
```

Comment:
You could use the `GordonDialogBox` for a cleaner JSX that's consistent with a lot of other dialogs across the site.

Reply:
Update Social Media Link is also not using GordonDialogBox. Should I use GordonDialogBox?

Reply:
I will make `GordonDialogBox` prettier and then come back to this issue and change it.

Now: 
![Screen Shot 2021-07-01 at 4 19 31 PM](https://user-images.githubusercontent.com/70544403/124184890-35536f80-da88-11eb-895d-a7e719f8ef28.png)

Change to:
![Screen Shot 2021-07-01 at 4 19 17 PM](https://user-images.githubusercontent.com/70544403/124184933-443a2200-da88-11eb-87b7-fc8c612a02ee.png)

or

<img width="536" alt="Screen Shot 2021-07-01 at 4 15 49 PM" src="https://user-images.githubusercontent.com/70544403/124184953-4a300300-da88-11eb-865a-dc7cd69eeb98.png">

Reply:
And also will work on Social Media Links to use `GordonDialogBox`.

Reply:
@EjPlatzer